client/src/OLD-App.js

```diff
@@ -0,0 +1,228 @@
+/*global chrome*/
+
+import React, { Component } from 'react';
+import moment from 'moment';
+import STATUSES from './timer-status';
+import './App.css';
+import TimerDisplay from './components/TimerDisplay';
+import TimerButton from './components/TimerButton';
+
+class App extends Component {
+  state = {
+    pomoDuration: moment.duration(6, 'seconds'),
+    shortBreakDuration: moment.duration(3, 'seconds'),
+    longBreakDuration: moment.duration(10, 'seconds'),
+    // countdownID: null,
+    // timerDisplay: moment.duration(6, 'seconds'),
+    // timerStatus: STATUSES.NOT_SET,
+    // pomoCounter: 0,
+    countdownID: chrome.extension.getBackgroundPage().timer.countdownID,
+    timerDisplay: chrome.extension.getBackgroundPage().timer.remaining,
+    timerStatus: chrome.extension.getBackgroundPage().timer.timerStatus,
+    pomoCounter: chrome.extension.getBackgroundPage().timer.pomoCount,
+    background: chrome.extension.getBackgroundPage()
+  };
+
+  componentDidMount() {
+    setInterval(
+      () =>
+        this.setState({
+          background: chrome.extension.getBackgroundPage()
+        }),
+      500
+    );
+  }
+
+  toggleTimer = () => {
+    const { background } = this.state;
+    if (this.state.timerStatus !== 'TIMER_RUNNING') {
+      background.timer.timerStatus = background.STATUSES.TIMER_RUNNING;
+      background.timer.countdownID = setInterval(this.reduceTimer, 1000);
+    } else {
+      background.timer.countdownID = clearInterval(
+        background.timer.countdownID
+      );
+      background.timer.timerStatus = background.STATUSES.TIMER_PAUSED;
+    }
+  };
+
+  // toggleTimer = () => {
+  //   if (this.state.timerStatus !== 'TIMER_RUNNING') {
+  //     this.setState({
+  //       timerStatus: STATUSES.TIMER_RUNNING,
+  //       countdownID: setInterval(this.reduceTimer, 1000)
+  //     });
+  //   } else {
+  //     this.setState({
+  //       timerStatus: STATUSES.TIMER_PAUSED,
+  //       countdownID: clearInterval(this.state.countdownID)
+  //     });
+  //   }
+  // };
+
+  resetTimer = () => {
+    const { background } = this.state;
+
+    background.timer.timerStatus = background.STATUSES.NOT_SET;
+    background.timer.countdownID = clearInterval(background.timer.countdownID);
```

Comment:
afaik clearInterval doesn't return anything in which case I think the assignment is unnecessary.
```suggestion
    clearInterval(background.timer.countdownID);
```